### PR TITLE
RFC: Removing the "Perf" and "Network" tabs from the in-app Element Inspector

### PR DESCRIPTION
**Pull request by huntie**
Identifies two sub-features of the in-app Element Inspector which we believe are **not worth reimplementing** under the New Architecture.

# [View the rendered RFC](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0777-remove-legacy-element-inspector-features.md)

### CODE REVIEW

File: proposals/0777-remove-legacy-element-inspector-features.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+author:
+- Alex Hunt <huntie@meta.com>
+date: 2024-04-04
+---
+
+# RFC0777: Removing the "Perf" and "Network" tabs from the in-app Element Inspector
+
+## Summary
+
+This RFC identifies two sub-features of the in-app Element Inspector which we believe are **not worth reimplementing** under the New Architecture.
```
Comment (cortinico) [inline proposals/0777-remove-legacy-element-inspector-features.md:12]: Q: Can you elaborate also on the other 2 tabs - Inspect and Touchables? Why are we keeping them? Are people actually using them?

**Review by cortinico — APPROVED** [review]
Accepting as in general I'm in favor of this removal 👍

**Comment by brentvatne** [comment]
this seems like a pragmatic decision to me. I doubt many people actually use these features, indeed I had forgotten these buttons even existed. there are better ways of getting both types of information.

**Review by elicwhite — APPROVED** [review]

**Comment by RayKay91** [comment]
What is a better way to get the info for the perf tab? I use this all the time!

**Comment by huntie** [comment]
@RayKay91 Some options for you in [this post](https://shift.infinite.red/why-you-dont-need-flipper-in-your-react-native-app-and-how-to-get-by-without-it-3af461955109#:~:text=alternatives%20for%20now.-,Log%20network%20calls,-Fetching%20network%20calls).

**Comment by matinzd** [comment]
I haven't even clicked once on this since the beginning.

File: proposals/0777-remove-legacy-element-inspector-features.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+As we build out the New Architecture, we need to consider the usefulness of these features and the cost of adapting and maintaining these against the Bridgeless model. So, we are considering the removal of both the **Perf** and **Network** features under the Element Inspector overlay, under the New Architecture.
+
+- Under the current architecture: Leave these features in place.
+- Under the New Architecture: Do not reimplement these features, removing the "Perf" and "Network" buttons.
+
+## Detailed design
+
+### Legacy features to remove
+
```
Comment (henryprigett5-creator) [inline proposals/0777-remove-legacy-element-inspector-features.md:40]: 1000

**Pull request merged**